vector/src/main/res/drawable/ic_shield_unknown.xml

```diff
@@ -0,0 +1,20 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+  <path
+      android:strokeWidth="1"
+      android:pathData="M12.008,23.487C12.005,23.487 12.002,23.488 12,23.489C11.998,23.488 11.995,23.487 11.992,23.487C11.92,23.471 11.813,23.445 11.675,23.409C11.399,23.337 11.002,23.223 10.523,23.058C9.565,22.725 8.292,22.186 7.022,21.361C4.49,19.715 2,16.954 2,12.405V3.45L12,0.521L22,3.45V12.405C22,16.954 19.51,19.715 16.978,21.361C15.708,22.186 14.435,22.725 13.477,23.058C12.998,23.223 12.601,23.337 12.325,23.409C12.187,23.445 12.08,23.471 12.008,23.487Z"
+      android:fillColor="#737D8C"
```

Comment:
Maybe define and use a new color for this new shield as per https://github.com/vector-im/element-android/blob/main/library/ui-styles/src/main/res/values/colors.xml#L142
Also update line 14 and file `ic_shield_unknown_no_border.xml`

Reply:
I have added the new color resource but I had to enable backward compatibility for API < 24 (see https://developer.android.com/develop/ui/views/graphics/vector-drawable-resources#vector-drawables-backward-solution). The update: [cf25b81](https://github.com/vector-im/element-android/pull/7361/commits/cf25b81e9aa420d284b841176e79eb907c8a3dad)